src/main/java/lotto/ui/InputView.java

```diff
@@ -19,7 +19,13 @@ public int inputPurchaseAmount() {
     public List<Integer> inputWinningNumbers() {
         System.out.println("지난 주 당첨 번호를 입력해 주세요.");
         String inputs = scanner.nextLine();
-        return Arrays.stream(inputs.split(",")).map(String::trim).map(input -> Integer.parseInt(input))
-                .collect(Collectors.toList());
+        return Arrays.stream(inputs.split(",")).map(String::trim)
+            .map(input -> Integer.parseInt(input))
```

Comment:
```suggestion
            .map(Integer::parseInt)
```
메소드 레퍼런스 문법을 이용하면 간결해져요~